backend/core/scripts/import-helpers.ts

```diff
@@ -159,6 +159,16 @@ export async function importListing(
   const relationsKeys = []
   listing = reformatListing(listing, relationsKeys)
 
+  // If a managementWebsite is provided, make sure it is a well-formed URL.
+  if (listing.managementWebsite) {
+    if (!listing.managementWebsite.startsWith("http")) {
+      listing.managementWebsite = "http://" + listing.managementWebsite
+    }
+
+    // This next line will throw an error if managementWebsite is a malformed URL.
+    new URL(listing.managementWebsite)
```

Comment:
We can't just remove it if it's not a well formed URL? Would it at least get logged as an error? I think as a user I might prefer that it succeed but not contain a malformed URL rather than fail to upload at all.

Reply:
SG about logging it as an error and just importing the listing without the URL!